portfolio/pom.xml

```diff
@@ -18,10 +18,20 @@
 
   <dependencies>
     <dependency>
-      <groupId>javax.servlet</groupId>
-      <artifactId>javax.servlet-api</artifactId>
-      <version>4.0.1</version>
-      <scope>provided</scope>
+        <groupId>javax.servlet</groupId>
```

Comment:
clang-format has fixed this too

Reply:
Did clang-format suggest 4 spaces here? I'm not sure if clang will change pom.xml files (it doesn't seem to mention pom.xml files in its documentation), so it might just be leaving it as you originally wrote it. I could be wrong though...